xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
 Linux,ou=devel,dc=example,dc=com</screen>
   </sect2>
  </sect1>
- <sect1 xml:id="sec.security.ldap.info">
+
+<sect1 xml:id="sec.security.ldap.yast.usergr">
+  <title>Configuring LDAP Users and Groups in &yast;</title>
+<!--taroth 2019-02-25: let's see where to move/integrate the content from this
+    section in the end-->
+  <para>
+   The actual registration of user and group data differs only slightly from
+   the procedure when not using LDAP. The following instructions relate to
+   the administration of users. The procedure for administering groups is
+   analogous.
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Access the &yast; user administration with <menuchoice>
+     <guimenu>Security and Users</guimenu><guimenu>User and Group
+     Management</guimenu></menuchoice>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Use <guimenu>Set Filter</guimenu> to limit the view of users to the
```

Comment:
This seems ambiguous. It depends on the pam module, but "limiting the view" could mean "limiting which users have the right to login to this system (authorisation)". I think this should be made clearer.

Reply:
Tried to rephrase and added one step. Hopefully clearer now.